webapp/src/modules/mapBuilder.js

```diff
@@ -224,18 +224,16 @@ export default class MapBuilder {
     const ridingRestrictionContent = `<h6 class='mb-0'>${t(
       "mobility:do_not_ride_title"
     )}</h6><p class='m-0'>${t("mobility:do_not_ride_intro")}</p>`;
-    const allRestrictionsContent =
-      parkingRestrictionContent + "<hr />" + ridingRestrictionContent;
-    if (restriction === "do-not-park") {
+    if (restriction.startsWith("do-not-park")) {
       popup.setContent(parkingRestrictionContent);
       polygonFillOpacity = 0.2;
     }
-    if (restriction === "do-not-ride") {
+    if (restriction.startsWith("do-not-ride")) {
       popup.setContent(ridingRestrictionContent);
       polygonFillOpacity = 0.2;
     }
-    if (restriction === "do-not-park-or-ride") {
-      popup.setContent(allRestrictionsContent);
+    if (restriction.startsWith("do-not-park-or-ride")) {
```

Comment:
Seems like this is pre-empted by line 227. This should be the first condition, and it should be using if/else.

Reply:
(you can copy the polygon fill opacity line a 3rd time, not a big deal)

Reply:
Thanks for catching that.

Reply:
The idea for polyfill was for overlapping layers to have a darker tint but it's totally unnecessary, I'll remove it.